CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ## [unreleased] (Added 🚀 | Changed | Removed 🗑 | Fixed 🐞 | Chore 👨‍💻 👩‍💻)
 
+### Fixed 🐞
+
+-   Fix ribbon-bar shadow moving down when search is expanded #3138
```

Comment:
Minor thing: 
Would you please check already existing changelog entries and the CONTRIBUTING.md on how to write changelog enries in CodeCharta?
In this case the link to the PR should be adjusted.


```suggestion
-   Fix ribbon-bar shadow moving down when search is expanded [#3138](https://github.com/MaibornWolff/codecharta/pull/3138)
```